### ISSUE TITLE
Not working on phoenix_live_view 0.20.4?

### ISSUE THREAD

**Issue by Cowa** [maxmarcon/live_select#58]
Hello!

With the update of LiveView `0.20.4`, live_select seems to no longer work.

When clicking on the input there is this error in the console:
`Uncaught TypeError: form is null`

When I downgrade LiveView to `0.20.3`, it's working.

https://github.com/maxmarcon/live_select/assets/1422403/f8ceba0a-2d94-4ddc-ab69-8ab2091fb034

**Comment by maxmarcon** [comment]
Hi @Cowa! Can you please show me how you're rendering live select in your code?

**Comment by Cowa** [comment]
I am rendering like this:
```elixir
<.form for={@form} phx-change="organization_selected">
  <.live_select
    field={@form[:organization_reference]}
    id="search_input"
  />
</.form>
```

`@form` is set like this: `assign(socket, :form, to_form(%{"organization_reference" => ""}))}`

**Comment by maxmarcon** [comment]
I see a different error:

<img width="595" alt="image" src="https://github.com/maxmarcon/live_select/assets/11156096/a1ac0443-6390-4d0a-bace-6e3abca3b1fe">

This is most likely a bug in live view JS stack. I suggest you open an issue there with a minimal repo that reproduces it. I'm sure Chris and Co will fix it in no time!

**Comment by maxmarcon** [comment]
by the way, if you do create an issue for LV, it would be great if you could linked it here. Thanks!

**Comment by maxmarcon** [comment]
looks like this is the issue: https://github.com/phoenixframework/phoenix_live_view/issues/3095

**Comment by Cowa** [comment]
Thanks for finding out it was a bug on LiveView.
The bug is fixed with LiveView 0.20.5, I close this issue!
